.changeset/green-rice-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
+---
+
+Use new recover upsell redirection
```

Comment:
For now this description is not accurate, same for LLD.
Your PR title would be more accurate 👍